contracts/interfaces/IRelayHub.sol

```diff
@@ -68,12 +68,6 @@ interface IRelayHub {
         bytes returnValue
     );
 
-    event Penalized(
```

Comment:
Why did you remove the event?

Reply:
We have never (in v2) actually emitted the event in the RelayHub. Penalize event is in the StakeManager as theoretically multiple hubs can cause a penalization